app/src/main/res/values-cht/string.xml

```diff
@@ -0,0 +1,112 @@
+    <string name="line_devilish_pervert">我從來沒想過你竟然這麼變態，真是看錯你了！</string>
+    <string name="line_i_guess">…也是。</string>
+    <string name="line_nice">nice!</string>
+    <string name="line_pervert_confirmed">變態確定！</string>
+    <string name="line_sorry">抱歉…</string>
+    <string name="line_sounds_tough">辛苦了呢。</string>
+    <string name="line_this_guy_hopeless">沒救了這傢伙，得快點做些什麼。</string>
+    <string name="line_gah">咔！</string>
+    <string name="line_dont_add_tina">去掉後面的「汀娜」！！</string>
+    <string name="line_pervert_idiot_wanttodie">你這個變態！！你是笨蛋嗎？想死嗎？</string>
+    <string name="line_pleased_to_meet_you">說起來，還沒正式自我介紹過呢。我是牧瀨紅莉栖，初次見面，請多關照。</string>
+    <string name="line_who_the_hell_christina">誰是克莉絲汀娜！！我才沒說過。</string>
+    <string name="line_why_christina">我很好奇，為什麼我叫做克莉絲汀娜？</string>
+    <string name="line_christina">克莉絲汀娜？？</string>
+    <string name="line_ask_me_whatever">什麼都可以問我，我會盡力回答的。</string>
+    <string name="line_could_i_help">那個，需要幫忙嗎？</string>
+    <string name="line_what_do_you_want">需要幫忙嗎？</string>
+    <string name="line_what_is_it">怎麼了呢？</string>
+    <string name="line_heheh">呵呵呵。</string>
+    <string name="line_huh_why_say">咦？ 為什麼這麼說？</string>
+    <string name="line_you_sure">是嗎？</string>
+    <string name="line_nice_to_meet_okabe">岡部倫太郎…初次見面，我是牧瀨紅莉栖。</string>
+    <string name="line_look_forward_to_working">請多指教。</string>
+    <string name="line_senpai_question">那麼前輩，我可以再問一個問題嗎？</string>
+    <string name="line_senpai_question_mark">前輩？！</string>
+    <string name="line_senpai_what_we_talkin">吶，前輩，剛剛的說的...</string>
+    <string name="line_senpai_who_is_this">那個…前輩，這位是？</string>
+    <string name="line_senpai_please_dont_tell">前輩！！拜託，無論如何都對其他人保密！</string>
+    <string name="line_still_not_happy">我不能接受…</string>
+    <string name="line_tm_nonsense">完全nonsense呢。</string>
+    <string name="line_tm_not_possible">這個嘛…，從結論來看，時間機器是不可能的。</string>
+    <string name="line_tm_scientist_no_evidence">可能只是因為科學家還沒發現某些關鍵要素，對吧？</string>
+    <string name="line_tm_we_dont_know">但是，也不能完全否定它的可能性。<string>
+    <string name="line_tm_you_said">你說…時間機器？</string>
+    <string name="line_humans_software">人們不也常說自己是由硬體和軟體組合成的嗎?</string>
+    <string name="line_memory_complex">然而，和一般的資料相比，記憶資料遠複雜得多。</string>
+    <string name="line_secret_diary">就像是，秘密的日記呢。</string>
+    <string name="line_modifying_memories_impossible">修改我的記憶？理論上是可行的。</string>
+    <string name="line_memories_christina">舉例來說，可以做到讓我以為自己的名字是克莉絲汀娜。</string>
+    <string name="line_gah_extended">咔、啊、啊啊啊...</string>
+    <string name="line_Leskinen_awesome">太棒了，他真的不簡單。</string>
+    <string name="line_Leskinen_nice">Nice!</string>
+    <string name="line_Leskinen_oh_no">Oh, No!</string>
+    <string name="line_Leskinen_shaman">哪裡！哪裡有日本巫女？！？倫太郎。</string>
+    <string name="line_Leskinen_holy_cow">天吶！所以我說我們該早點來的。</string>
+    <string name="line_should_christina">還是說，我該介紹自己是克莉絲汀娜比較好呢？</string>
+    <string name="line_ok">什麼？</string>
+
+    <!-- Input strings -->

```

Comment:
Including input strings is unnecessary; they are read from the selected recognition language (in settings)